doc/index.rst

```diff
@@ -16,43 +16,60 @@ Mailadm keeps all configuration, token and user state in a single
 sqlite database.  It comes with an example install script that
 can be modified for distributions.
 
-.. note::
 
-    At this point installation is only well supported/documented for particular
-    Dovecot/Postfix/Systemd/Nginx configurations. The mailadm python software does
-    not depend on it much but you will have to figure the use of other server software
-    yourselve. It might still make sense to use part of the install script.
-    As we are advocating working from a git checkout you may keep your own
-    branch in your local git and update with remote master from time to time.
-    Please report bugs and improvements as issues or PR requests.
+Quick Start
+-----------
 
+.. note::
 
-Quickstart
-----------
+    To use mailadm, you need admin access to a `Mailcow
+    <https://mailcow.email/>`_ instance. You can run mailadm as a docker
+    container, either on the same machine as mailcow or somewhere else.
 
-Get a git copy of the mailadm repository and change into it.
+First get a git copy of the mailadm repository and change into it.
 
     $ git clone https://github.com/deltachat/mailadm
     $ cd mailadm
 
+Now you need to configure some environment variables in a file called ``.env``:
 
-Now **review and then run** this install script:
+* ``MAIL_DOMAIN``: the server name of the email addresses your users will have.
+* ``WEB_ENDPOINT``: the web endpoint of mailadm; make sure mailadm receives
+  POST requests at this address.
+* ``MAILCOW_ENDPOINT``: the API endpoint of your mailcow instance.
+* ``MAILCOW_TOKEN``: the access token for the mailcow API; you can generate it
+  in the mailcow admin interface.
 
-    $ sudo bash install_mailadm.sh
+In the end, your ``.env`` file will probably look like this:
 
-Note that you can call this command repetetively if you want
-to play around. By default this script:
+.. code:: bash
 
-- creates a `mailadm` user and install the mailadm software into it (from
-  the checkout, not via the Python package index)
+    export MAIL_DOMAIN=mail.example.org
+    export WEB_ENDPOINT=http://mailadm.example.org:3691/
+    export MAILCOW_ENDPOINT=https://mailcow-web.example.org/api/v1/
+    export MAILCOW_TOKEN=932848-324B2E-787E98-FCA29D-89789A
+    
+Now you can build and run the docker container:
 
-- creates config files at approprirate system locations
-  for integrating with systemd, dovecot, postfix and nginx.
+.. code:: bash
 
-- creates a `~mailadm/README.txt` file with further instructions
-  how to do 2-3 line changes to your existing nginx and dovecot
-  config files.
+    $ sudo docker build . -t mailadm-mailcow
+    $ touch mailadm.db
+    $ . .env && sudo docker run --mount type=bind,source=$PWD/mailadm.db,target=/mailadm.db --mount type=bind,source=$PWD/.env,target=/mailadm/.env --rm mailadm-mailcow /usr/bin/mailadm init --web-endpoint $WEB_ENDPOINT --mail-domain $MAIL_DOMAIN --mailcow-endpoint $MAILCOW_ENDPOINT --mailcow-token $MAILCOW_TOKEN
```

Comment:
are there sometimes other db-related files like mailadm.wap or .mailadm.db or something that might need binding? 
Might make anyway sense to rather create a `docker-data` directory (with .env and mailadm.db files) and bind-mount that into docker?  It's better than creating it in-place in the main dir of the checkout.  Also the .gitignore can then ignore the `docker-data` directory.

Reply:
makes sense, but let's do this after this PR :) I put it into #27.